### PR TITLE
userInfoProfile component created

### PR DESCRIPTION
**Pull request by DexHarris**

### CODE REVIEW

File: components/tabsUserProfile/userInfoProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+const userInfoProfile = () => {
+    return (
+        <>
+            <div className="p-6 flex flex-row">
+                <img className="bg-slate-600 w-32 h-36 mt-3 mr-10 rounded-lg drop-shadow-[0_10px_8px_rgba(0,0,0,0.15)]"></img>
+                <div className="flex flex-col mt-2">
+                    <h1 className="font-bold text-3xl">Italo Melo</h1>
```
Comment (moncapitaine) [inline components/tabsUserProfile/userInfoProfile.tsx:8]: h2 is better?

File: components/tabsUserProfile/tabProfile.tsx [hunk 2]
```diff
@@ -1,8 +1,10 @@
+import UserInfoProfile from "./userInfoProfile"
+
 export const TabProfile = () => {
   return (
-    <div className="flex min-h-[600px]">
+    <div className="flex min-h-[50vh] px-2 mx-2">
       <div>
```
Comment (moncapitaine) [inline components/tabsUserProfile/tabProfile.tsx:6]: remove div

File: components/tabsUserProfile/tabFollowers.tsx [hunk 3]
```diff
@@ -4,22 +4,22 @@ import SearchRoundedIcon from '@mui/icons-material/SearchRounded'
...
+      <div className="bg-gray-300 px-6 py-2 w-full flex items-center rounded-full">
+        <input
+          className="text-lg w-full bg-gray-300 outline-none"
+          type="text"
+          placeholder="Search for Follower"
+        />
+        <SearchRoundedIcon />
       </div>
-    </>
+      <div className="mt-2">
```
Comment (moncapitaine) [inline components/tabsUserProfile/tabFollowers.tsx:16]: remove div?

Comment (moncapitaine) [reply components/tabsUserProfile/tabFollowers.tsx:16]: instead of mt, use gap on parent

File: components/tabsUserProfile/tabFollowers.tsx [hunk 4]
```diff
@@ -4,22 +4,22 @@ import SearchRoundedIcon from '@mui/icons-material/SearchRounded'
...
-          />
-          <SearchRoundedIcon />
-        </div>
-        <div className="mt-2">
-          <ul>
-            <li>Follower 1</li>
-            <li>Follower 2</li>
-          </ul>
-        </div>
+    <div className=''>
```
Comment (moncapitaine) [inline components/tabsUserProfile/tabFollowers.tsx:7]: simplify.

File: pages/userProfile.page.tsx [hunk 5]
```diff
@@ -1,155 +1,86 @@
...
-import { PreferredIngredients } from '../components/preferredIngredients/preferredIngredients'
 import { useUserProfile } from '../context/useUserProfile'
 import { Tab } from '@headlessui/react'
 import { TabOwnRecipes } from '../components/tabsUserProfile/tabOwnRecipes'
 import { TabFollowers } from '../components/tabsUserProfile/tabFollowers'
 import { TabProfile } from '../components/tabsUserProfile/tabProfile'
-import { TabSubscriptions } from '../components/tabsUserProfile/tabSubscriptions'
 
 function classNames(...classes: any[]) {
   return classes.filter(Boolean).join(' ')
```
Comment (moncapitaine) [inline pages/userProfile.page.tsx:9]:
```suggestion
  return classes.join(' ')
```

File: components/tabsUserProfile/tabFollowers.tsx (hunk 3 above)
Comment (DexHarris) [reply components/tabsUserProfile/tabFollowers.tsx:16]: done

File: components/tabsUserProfile/tabProfile.tsx (hunk 2 above)
Comment (DexHarris) [reply components/tabsUserProfile/tabProfile.tsx:6]: done

**Review by moncapitaine — APPROVED** [review]

**Review by Bhoft — APPROVED** [review]

**Pull request merged**